js/models/certification.ts

```diff
@@ -29,7 +30,20 @@ export const isExpired = (c: Certification, now: DateTime) => {
   return now >= c.expires.startOf("day");
 };
 
-export const getExpired = (cs: Certification[] | undefined, now: DateTime) => {
+export const filterRelevantForOperators = (
+  cs: Certification[],
+  line: HeavyRailLine,
+) => {
+  return cs.filter(
+    (c) =>
+      (c.type === "rail" && c.railLine === line) || c.type === "right_of_way",
```

Comment:
Do ROW certs have a line associated with them? Can you use an OL ROW cert on the BL? (I don't think they have separate ROW trainings by line, so I think this is working correctly, but wanna double check.)

Also there's no test for whether an OL ROW cert can apply to BL (all other cases are covered tho, and if you think that test isn't needed that's fine.)

Reply:
ROW certs are line agnostic, that's correct, so that case is really undefined behavior. _However_, because I tested the hypothetical case whether a blue ROW cert exists, I should probably test the case where an unmatching-line ROW cert exists and is ignored.